examples/replica_exchange/run_replica_exchange.py

```diff
@@ -85,7 +88,7 @@
 }
 
 # Torsion angle definitions
-torsion_force_constant = 0.2 * unit.kilocalorie_per_mole / unit.radian / unit.radian
+torsion_force_constant = 20.0 * unit.kilocalorie_per_mole / unit.radian / unit.radian
```

Comment:
Periodic torsion units should be kcal/mol?

Reply:
Yep, didn't fix it here.